package.json

```diff
@@ -80,8 +80,8 @@
     "@types/testing-library__cypress": "^5.0.5",
     "@types/ua-parser-js": "^0.7.35",
     "@types/wcag-contrast": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.1.0",
-    "@typescript-eslint/parser": "^4.1.0",
+    "@typescript-eslint/eslint-plugin": "^5.19.0",
```

Comment:
just for completeness, why is this version bump required? as you can see from the yarn.lock we're now installing 2 major versions of these 2`@typescript-eslint` packages, which is a bit unfortunate (though i know it only affects local dev builds not prod code)